install_config/configuring_red_hat_registry.adoc

```diff
@@ -0,0 +1,127 @@
+[[install-config-configuring-red_hat_registry]]
+= Accessing and Configuring the Red Hat Registry
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[auth-enabled-registry]]
+== Authentication Enabled Red Hat Registry
+
+include::architecture/topics/auth_enabled_registry.adoc[leveloffset=+1]
+
+
+[[create-authentication-tokens]]
+=== Creating Service Accounts and Authentication Tokens
+
+You must create tokens if your organization manages shared accounts.
+Administrators can create, view, and delete all tokens associated with an
+organization.
+
+//[[Prerequisites]]
+*Prerequisites*
+
+* User credentials
+
+//[[Procedure]]
+*Procedure*
+
+To create a token in order complete a `docker login`:
+
+. Navigate to `registry.redhat.io`.
+. Log in with your RHN username and password.
+. Accept terms when prompted.
+.. If you are not immediately prompted to accept terms, you will be prompted
+when proceeding with the following steps.
+. From the *Registry Service Accounts* page, click *Create Service Account*
+.. Provide a name for the service account. It will be prepended with a random string.
+.. Enter a description.
+.. Click create.
+. Navigate back to your Service Accounts.
+. Click the Service Account you created.
+. Copy the username, including the prepended string.
+. Copy the token.
+
+[[using-service-accounts]]
+=== Using Service Accounts
+
+Once you have created your service accounts and generated tokens, you can perform
+additional tasks.
+
+[NOTE]
+====
+This section provides the manual steps, which can be automatically performed
+during installation.
```

Comment:
Does this mean that if you provide the relevant inventory parameters and values that you don't have to do this? I'd be a little more explicit.